pkg/cluster/bootstrap.go

```diff
@@ -52,8 +52,8 @@ func (c *Cluster) Bootstrap(ctx context.Context, snapshot bool) error {
 			// instance of etcd in the event that etcd certificates are unavailable,
 			// reading the data, and comparing that to the data on disk, all the while
 			// starting normal etcd.
-			isHTTP := c.config.JoinURL != "" && c.config.Token != ""
-			if isInitialized && !isHTTP {
+			if isInitialized {
```

Comment:
This appears to be changing behavior from 
Original: If on a "secondary" node, don't start a temporary etcd with what's on disk
New: On every node, start a temporary etcd to grab bootstrap data out of it

Does this not pose a problem if etcd was stopped on a secondary node and changes to bootstrap data were made on the "live" cluster? 

I'm thinking if possible we should attempt to reconcile bootstrap data out of the live etcd cluster (because our local etcd member may actually be out of date as it was not running, but will catch up once it starts) and if that fails b/c the etcd cluster is dead, fall back to trying to reconcile from the local files.

To be fair, a similar issue like what I'm thinking about was possible to hit with the init node with the original behavior.

Reply:
This is not an issue, reconciliation does occur even after the rest of the live cluster has changed the bootstrap data.